api/renter.go

```diff
@@ -489,3 +633,19 @@ func (api *API) renterUploadHandler(w http.ResponseWriter, req *http.Request, ps
 	}
 	WriteSuccess(w)
 }
+
+// stringToBool converts "true" and "false" strings to their respective boolean value and returns an error if conversion is not possible.
+func stringToBool(param string) (bool, *Error) {
+	// Parse the async parameter.
+	var out bool
+	switch {
+	case param == "true":
+		out = true
+	case len(param) == 0 || param == "false":
+		out = false
+	default:
+		return false, &Error{"asyncparam has to be empty, \"true\" or \"false\""}
```

Comment:
error needs to be more generic, perhaps "boolean parameters need to take the form "true" or "false""